pkg/api/types.go

```diff
@@ -341,16 +341,50 @@ type TestStepConfiguration struct {
 	OpenshiftInstallerCustomTestImageClusterTestConfiguration *OpenshiftInstallerCustomTestImageClusterTestConfiguration `json:"openshift_installer_custom_test_image,omitempty"`
 }
 
+// RegistryReferenceConfig is the struct that step references are unmarshalled into
+type RegistryReferenceConfig struct {
+	Reference RegistryReference `json:"step,omitempty"`
+}
+
+// RegistryReference contains the LiteralTestStep of a reference as well as the documentation for the step
+type RegistryReference struct {
+	LiteralTestStep `json:",inline"`
+	Documentation   string `json:"documentation,omitempty"`
```

Comment:
In general these are the actual types that we will expect people to be filling in when they make PRs to add new files to the registry, so we will link them the GoDoc page to read when they have questions on how to configure something, so we always want doc on what exported fields are for, too

Reply:
(and let's make sure our comments here are written for that audience)